docs/ops.html

```diff
@@ -2113,6 +2113,22 @@ <h5 class="anchor-heading"><a id="kraft_controller_monitoring" class="anchor-lin
     <td>The number of active brokers as observed by this Controller.</td>
     <td>kafka.controller:type=KafkaController,name=ActiveBrokerCount</td>
   </tr>
+  <tr>
+    <td>Migrating ZK Broker Count</td>
+    <td>The number of brokers registered with the Controller that haven't yet migrated to KRaft mode.</td>
+    <td>kafka.controller:type=KafkaController,name=MigratingZkBrokerCount</td>
+  </tr>
+  <tr>
+    <td>ZK Migrating State</td>
+    <td>
+      0 - NONE, cluster created in KRaft mode;
+      4 - ZK, Migration has not started, controller is a ZK controller.
+      2 - PRE_MIGRATION, the KRaft Controller is waiting for all ZK brokers to register in migration mode;
+      1 - MIGRATION, ZK metadata has been migrated, but some broker is still running in ZK mode;
+      3 - POST_MIGRATION, the cluster migration is complete;
```

Comment:
https://github.com/apache/kafka/blob/3617dda9a5415ee1597967f754985fb73a5350c6/metadata/src/main/java/org/apache/kafka/metadata/migration/ZkMigrationState.java#L33-L72

Reply:
This is not going to render well on the website. HTML does not preserve new lines.

Reply:
That's true, I thought it wouldn't matter, but on second thought there's no reason we can't present a bit more nicely. Updated.

Reply:
If you want to see how it renders, you can:
- run `./gradlew siteDocsTar`
- clone [apache/kafka-site](https://github.com/apache/kafka-site)
- extract `core/build/distribution/kafka*-site-docs.tgz` and copy its content to `kafka-site/3.7`
- run `./start-preview.sh` in `kafka-site` and open localhost:8080

Reply:
Thanks @mimaison, that's useful. Here's what it looks like:
<img width="683" alt="image" src="https://github.com/apache/kafka/assets/1357510/f5ac55bc-2262-40fd-8913-9a83deb4e543">

Reply:
Great thanks!
Nit: do you want to put a semi colon after `ZK, Migration has not started, controller is a ZK controller` instead of the dot?

Reply:
Well spotted, that was unintended. Updated